Test/Test_IMU.c

```diff
@@ -0,0 +1,331 @@
+#include <stdio.h>
+#include "cmsis_os.h"
+#include "stm32f4xx_hal.h"
+#include "main.h"
+#include "IMU.h"
+/* TEST PROCEDURE:
+    1: Make proper connections between main board and IMU (Although the INT and RESET pins are connected, they are not
+    used because we do not use low power mode)
+    2: Follow steps in Pg. 51 of datasheet for calibration of all 3 sensors
+    3: It is a bit difficult to test since you would have to be moving your laptop. Since the magnetometer is also 
+    initialized, we will test to see if we get readings from that (So get a fridge magnet and wave it around the 
+    IMU and check to see if the data changes). According to the datasheet, it uses the magnetometer to read gravity
+    as well so there should be some data already being sent.
+    4: If that works, then move the IMU around very quickly. This should change the accelerometer and gyro readings.
+    5: According to the data collected, determine if we should initialize the IMU in another mode. (NDOF mode is 
+    one to consider. Page 25 of datasheet)
+    6: If readings are wrong, use calibration registers to make them more accurate. (Registers 0x55-0x6A)
+*/
+I2C_HandleTypeDef hi2c1;
+UART_HandleTypeDef huart3;
+
+CAN_HandleTypeDef hcan1;
+
+I2C_HandleTypeDef hi2c2;
+
+SPI_HandleTypeDef hspi5;
+
+UART_HandleTypeDef huart1;
+
+HAL_StatusTypeDef error;
+
+IMUCalibData_t Calib_data;
```

Comment:
This shouldn't be a global (and shouldn't be put here with the hardware handles). Should probably be inside the task (inside `IMUTest()`).